src/main/resources/com/google/api/codegen/php/discovery_fragment.snip

```diff
@@ -248,8 +248,8 @@
   optParams
 @end
 
-@private requestBodyName()
-  requestBody
+@private postBodyName()
+  postBody
```

Comment:
Are there actual errors caused by this variable name? I looked everywhere for documentation of such and found nothing.